lib/cc/engine/fingerprint.rb

```diff
@@ -33,7 +33,13 @@ def compute
       attr_reader :path, :cop_name, :message
 
       def stripped_message
-        message.gsub(/ \[.+\]$/, "")
+        # example full message with details:
+        # Metrics/MethodLength: Method has too many lines. [18/10] (https://github.com/bbatsov/ruby-style-guide#short-methods)
+        url_regex = %r{ \(https\://github\.com.+\)}
```

Comment:
In all of the examples I saw it points to 
https://github.com/bbatsov/ruby-style-guide

Double checking the RuboCop source code.